app/src/main/java/com/github/andrezimmermann/gluappsample/server/api/GluApi.java

```diff
@@ -0,0 +1,107 @@
+package com.github.andrezimmermann.gluappsample.server.api;
+
+
+import com.github.andrezimmermann.gluappsample.server.api.command.FindByNameCommand;
+import com.github.andrezimmermann.gluappsample.server.api.command.FindDeparturesByRouteId;
+import com.github.andrezimmermann.gluappsample.server.api.command.FindStopsByRouteId;
+import com.github.andrezimmermann.gluappsample.server.converter.DataConverter;
+import com.github.andrezimmermann.gluappsample.server.converter.JsonConverter;
+import com.github.andrezimmermann.gluappsample.server.data.FindDeparturesByRouteIdParameter;
+import com.github.andrezimmermann.gluappsample.server.data.FindDeparturesByRouteIdResponse;
+import com.github.andrezimmermann.gluappsample.server.data.FindRouteByNameParameter;
+import com.github.andrezimmermann.gluappsample.server.data.FindRouteByNameResponse;
+import com.github.andrezimmermann.gluappsample.server.data.FindStopsByRouteIdParameter;
+import com.github.andrezimmermann.gluappsample.server.data.FindStopsByRouteIdResponse;
+import com.github.andrezimmermann.gluappsample.shared.data.BusDeparture;
+import com.github.andrezimmermann.gluappsample.shared.data.BusLine;
+import com.github.andrezimmermann.gluappsample.shared.data.BusStop;
+import com.github.andrezimmermann.gluappsample.shared.error.ServiceException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Class that maps the REST calls
+ */
+public class GluApi {
+
+    private DataConverter converter = new JsonConverter();
+
+
+    public List<BusLine> getRouteIdByName(String routeName) throws ServiceException {
+        FindByNameCommand command = new FindByNameCommand(Endpoint.FIND_ROUTE_ID, converter);
+        FindRouteByNameParameter parameter = new FindRouteByNameParameter(routeName);
+        FindRouteByNameResponse responseData = command.sendData(parameter);
+        List<FindRouteByNameResponse.FindRouteByNameRow> rows = responseData.getRows();
```

Comment:
You could change the command class to return a list of `BusLine` directly instead of returning the inner class type and implementing a method to convert the result.